src/kirby/components/list/list.component.scss

```diff
@@ -17,19 +16,33 @@ ul {
     p {
       margin: 0;
     }
-    
-    
 
     &.row{
-      &:first-of-type {
-        border-top: 1px solid get-color('kirby-grey-2');
+      border-color: get-color('kirby-grey-2');
+      border-style: solid;
+      border-width: 0 0 1px 0;
+
+      &.rounded {
+        border-width: 0 1px 1px 1px;
+
+        &.top {
```

Comment:
:fire: web already has `top`/`bottom` build into css pseudo selectors (`first-child`/`last-child`, `first-of-type` etc), so it shouldn't be neccesary to implement lowest common denominator (aka {N} ;) )  for web

Reply:
Please see the comment I made on `src/kirby/components/list/list.component.ts`